apps/google-analytics-4/frontend/src/locations/Sidebar.tsx

```diff
@@ -1,17 +1,13 @@
-import React from 'react';
-import { Paragraph } from '@contentful/f36-components';
-import { SidebarExtensionSDK } from '@contentful/app-sdk';
-import { /* useCMA, */ useSDK } from '@contentful/react-apps-toolkit';
+import SidebarAnalytics from 'components/sidebar/SidebarAnalytics';
 
 const Sidebar = () => {
-  const sdk = useSDK<SidebarExtensionSDK>();
   /*
      To use the cma, inject it as follows.
      If it is not needed, you can remove the next line.
   */
   // const cma = useCMA();
 
-  return <Paragraph>Hello Sidebar Component (AppId: {sdk.ids.app})</Paragraph>;
+  return <SidebarAnalytics />;
```

Comment:
Love that you made this into its own component! Great separation of concerns here. I am wondering if its possible this main analytic app content, like the dropdown, chart etc, will ever exist somewhere else besides the `sidebar`? I have absolutely no idea around the context of if this app could exist in other locations, but I get nervous about pigeon holing the file structure, and placement of components in directories based on their location in the app i.e sidebar directory. What are your thoughts?

Reply:
I was following the pattern of the `ConfigScreen` components being in a directory called `config-screen`, so since these are in the `Sidebar` I called the directory and component `sidebar`. I am happy to rename the `sidebar` directory to `main-app` and rename this component from `SidebarAnalytics` to `AnalyticsApp`, does that sound good?

Reply:
Yeah the logic totally makes sense. I think for now perhaps `sidebar` suffices as the directory name since that is the immediate location, and we can adjust if needed!